src/services/Comment.service.js

```diff
@@ -0,0 +1,45 @@
+import db from '../models';
+import AppError from '../utils/error';
+
+/**
+ * Class Comment services creates a middleware
+ */
+class CommentService {
+  /**
+   * findUserByEmail - used to get user from database by email
+   * @param {string} userId - email of the user
+   * @param {string} requestId - email of the user
+   * @param {string} comment - email of the user
+   * @returns {object} user data from database
+   */
+  async postRequestComment(userId, requestId, comment) {
+    const requestData = await db.request.findOne({ where: { id: requestId } });
+
+
+    if (!requestData) {
+      throw new AppError('The request with the ID provided is not available.', 404);
```

Comment:
I think you can change error message to: `request resource not found`

Reply:
Fixed.

Reply:
> I think you can change error message to: `request resource not found`

Fixed.